<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QMainWindow" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>733</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>733</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="saveAsMenu">
     <property name="title">
      <string>Save as ...</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="saveAsMenu"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Home</string>
    </property>
    <widget class="QMenu" name="menuThemes">
     <property name="title">
      <string>Themes</string>
     </property>
     <addaction name="actionDark"/>
     <addaction name="actionLight"/>
    </widget>
    <addaction name="menuThemes"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;About</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>120</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>300</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Paint Tool</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="penBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-pen-24.png</normaloff>:/images/icons8-pen-24.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="shortcut">
      <string notr="true"/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="brushBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Brush&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-cosmetic-brush-52.png</normaloff>:/images/icons8-cosmetic-brush-52.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="eraserBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Eraser&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-erase-48.png</normaloff>:/images/icons8-erase-48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bucketBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>50</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Fill Bucket&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-fill-color-52.png</normaloff>:/images/icons8-fill-color-52.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="lineBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Line&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-line-48.png</normaloff>:/images/icons8-line-48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="rectBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Rectangle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-rectangle-96.png</normaloff>:/images/icons8-rectangle-96.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="circleBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Circle&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-circle-48.png</normaloff>:/images/icons8-circle-48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="colorBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Color Panel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-color-mode-48.png</normaloff>:/images/icons8-color-mode-48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="widthBtn">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Width size&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/width.png</normaloff>:/images/width.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="clearBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>46</width>
       <height>34</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Clear screen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/images/icons8-cls-50.png</normaloff>:/images/icons8-cls-50.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/clear.png</normaloff>:/images/clear.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-exit-64.png</normaloff>:/images/icons8-exit-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-undo-64.png</normaloff>:/images/icons8-undo-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-redo-64.png</normaloff>:/images/icons8-redo-64.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-save-48.png</normaloff>:/images/icons8-save-48.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-cosmetic-brush-52.png</normaloff>:/images/icons8-cosmetic-brush-52.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Brush</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/line.png</normaloff>:/images/line.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Line</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/rect.png</normaloff>:/images/rect.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Ellipse</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/palette.png</normaloff>:/images/palette.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Color</string>
   </property>
   <property name="toolTip">
    <string>Select color to draw</string>
   </property>
  </action>
  <action name="actionWidth">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/width.png</normaloff>:/images/width.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Width</string>
   </property>
   <property name="toolTip">
    <string>Set line width</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/info.png</normaloff>:/images/info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-add-file-64.png</normaloff>:/images/icons8-add-file-64.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-single-page-mode-48.png</normaloff>:/images/icons8-single-page-mode-48.png</iconset>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
   <property name="toolTip">
    <string>Dark Mode</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/images/icons8-single-page-mode-light-48.png</normaloff>:/images/icons8-single-page-mode-light-48.png</iconset>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
   <property name="toolTip">
    <string>Light Mode</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
